### ISSUE TITLE
Add Cohort role, Program,  Member Attributes Type, Attributes Type, Cohort Type- No character limit is available on Enter Role Name field 

### ISSUE THREAD

**Issue by zohaibihs** [OpenSRP/openmrs-module-cohort#15]
Add Cohort role - No character limit is available on Enter Role Name field (There should be a character limit of 20 characters on text fields).

**Comment by bholagabbar** [comment]
Text limit is 255 characters as discussed with @maimoonak. Please tell me if it to be changes. Closed similar tickets and clubbed into one

**Comment by zohaibihs** [comment]
It need to be change as 255 characters for text fields is too much, like 20 characters should be allowed because it is not a text box to accept 255 characters.

**Comment by bholagabbar** [comment]
@maimoonak You said 20 was too less. What should the new limit be is 255 is too much?

**Comment by maimoonak** [comment]
I would say it should be more than 100. 100-150 is a safe limit.

**Comment by bholagabbar** [comment]
Should we keep 120 then

**Comment by zohaibihs** [comment]
As per discussion with Maimoona, it should be 100 characters.

**Comment by bholagabbar** [comment]
@maimoonak I'm not able to determine where I should be making the changes. Could you help me out? Which file(s) needs to be modified. I'm pretty sure it's not liquibase.xml. Is it in the hibernate mappings?

**Comment by maimoonak** [comment]
Is this constraint implemented on DB level? If you cant  find it in liquibase and hbms then I think its just on UI level. I would suggest to keep it as it is on DB level and donot make any change to default string len which is  255 chars! Just implement in Validator in Controller to limit input to 100 characters. You can also limit by adding size or maxlen attribute of html on control itself.

**Comment by bholagabbar** [comment]
Ok I will make the change in the HTML form

**Comment by bholagabbar** [comment]
Fixed in commit https://github.com/OpenSRP/openmrs-module-cohort/commit/b2f930448046f10ac1d896bef281939b668e77e6 . Also, hope we can close the issues with tags 'resolved' soon

**Comment by maimoonak** [comment]
Yeh. You can mark all resolved issues with tag 'resolved'. QA team would
Close the issue after confirmation in next release

**Comment by zohaibihs** [comment]
Cohort member role - No character limit is available on Enter Role Name field (There should be a character limit of 100 characters on text fields).

**Comment by bholagabbar** [comment]
@zohaibihs just issues a commit which fixes that 7bae54e61dae10e63c5d611c6c0773d2ff037c74

**Comment by zohaibihs** [comment]
Yes, I have seeing your code but its not working fine on front end.
Cohort member role - No character limit is available on Enter Role Name field. 

![image](https://cloud.githubusercontent.com/assets/20788534/17806958/0af57dcc-6621-11e6-88be-73930be8fa3c.png)

**Comment by bholagabbar** [comment]
I JUST issued it. It's fixes If you recompile the module
